app/models/user.rb

```diff
@@ -1952,6 +1960,15 @@ def check_site_contact_username
     end
   end
 
+  def sidebar_tags
+    return custom_sidebar_tags if custom_sidebar_tags.present?
```

Comment:
Is there a reason we're returning a scope instead of the tag names?

Reply:
yes, we need to have full objects as serializers is including information if tag is `pm_only`. That is used to create correct link

Reply:
Ahh ok :+1: